pkgs/development/libraries/kdiagram/default.nix

```diff
@@ -1,19 +1,18 @@
 {
-  mkDerivation, fetchurl, lib,
+  mkDerivation, fetchFromGitLab, lib,
   extra-cmake-modules, qttools,
   qtbase, qtsvg,
 }:
 
-let
+mkDerivation rec {
   pname = "kdiagram";
   version = "2.7.0";
-in
-
-mkDerivation {
-  inherit pname version;
-  src = fetchurl {
-    url = "https://download.kde.org/stable/${pname}/${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256:1pgvf2q8b59hw0jg5ajmj5nrn4q8cgnifpvdd0fynk2ml6zym8k3";
+  src = fetchFromGitLab {
```

Comment:
Why aren't we using the official release?

Reply:
The commit message of the unsquashed PR explains it - From some reason I couldn't find the latest git tag in that URL, maybe they forgot to push the tarball :man_shrugging: .